package.json

```
@@ -50,9 +50,10 @@
   "devDependencies": {
     "@eslint/js": "9.20.0",
     "@types/node": "^22.13.4",
+    "@types/react-router-dom": "^5.3.3",
```

```suggestion
    "@types/react-router-dom": "5.3.3",
```

I see that the types are also using a `^`, I think we should lock this dependency too.